vault/logical_system.go

```diff
@@ -2414,4 +2520,21 @@ This path responds to the following HTTP methods.
 		"Lists the headers configured to be audited.",
 		`Returns a list of headers that have been configured to be audited.`,
 	},
+	"plugin-catalog": {
+		`Configures the plugins known to vault`,
+		`
+This path responds to the following HTTP methods.
+    GET /
```

Comment:
Shouldn't this be a ListOperation and have the parameter be required instead of optional? This feels unexpected.